core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.html

```diff
@@ -94,7 +107,7 @@
     </div>
   </div>
 
-  <div ng-if="recommendedExplorationSummaries !== null && (collectionSummary || recommendedExplorationSummaries.length > 0)"
+  <div ng-if="!collectionSummary && recommendedExplorationSummaries !== null && recommendedExplorationSummaries.length > 0"
```

Comment:
In this div, remove the ones pertaining to collections as we don't need that there anymore.

Reply:
Done